gapic-generator-cloud/lib/google/gapic/generators/cloud_generator.rb

```diff
@@ -36,6 +36,8 @@ def initialize api
           use_helpers! :gem_presenter
         end
 
+        # rubocop:disable all
```

Comment:
This change broke both the `Metrics/AbcSize` and the `Metrics/MethodLength` cops. When this method grows it will break other cops as well. I find it easier to disable all cops rather than adding additional disable comments for every cop that breaks when changes are made.

We _could_ break this method up, but I really like having it all inline. I don't think adding any additional abstractions around this method adds to understanding what it is doing.

Reply:
What about just changing the rubocop configuration?

Reply:
I agree that it's best to keep a method like this in one piece.

Reply:
Right now we use the same rubocop config file for formatting the generated files and for validating the generator's code style. If we add exceptions to the generator's code style then we should use separate configuration files. But I can see this method continuing to grow, and I don't see much value is changing the rubocop config for this one method. I would prefer to treat this method as an exception.

Reply:
Ack. I think thats good enough reason for the catchall. Maybe it would be good to add a comment above that provides context to it. Consider this resolved on my end :+1: